src/rewriter/mod.rs

```diff
@@ -558,6 +649,110 @@ mod tests {
         assert_eq!(res, "<span>?</span>");
     }
 
+    #[test]
+    fn test_rewrite_adjust_charset_on_meta_tag_attribute_charset() {
+        use crate::html_content::{ContentType, TextChunk};
+
+        let enthusiastic_text_handler = || {
+            doc_text!(move |text: &mut TextChunk| {
+                let new_text = text.as_str().replace('!', "!!!");
+                text.replace(&new_text, ContentType::Text);
+                Ok(())
+            })
+        };
+        let html: Vec<u8> = [
```

Comment:
Here and in all other places: separate variable declarations, multiline expressions and statements by empty lines

Reply:
:heavy_check_mark: